Source/WebInspectorUI/UserInterface/Views/InlineSwatch.js

```diff
@@ -96,7 +96,7 @@ WI.InlineSwatch = class InlineSwatch extends WI.Object
             }
 
             this._swatchElement.addEventListener("click", this._swatchElementClicked.bind(this));
-            if (this._type === WI.InlineSwatch.Type.Color)
+            if (this._type === WI.InlineSwatch.Type.Color && this._allowChangingColorFormats())
```

Comment:
I think we could/should move `this._type === WI.InlineSwatch.Type.Color` inside `_allowChangingColorFormats` so that it's all right there (i.e. doesn't make sense to change color formats when `WI.InlineSwatch.Type.Bezier:` 😅)

NIT: This also prevents developers from switching from RGB to HSL.  Do we wanna restrict to RGB only?

Reply:
This will actually restrict to the format of the `value` provided. In the case of the the overlay color swatches, that's `HSL`, which is what is serialized to disk and reloaded in the future anyways.